### PR TITLE
managedvolume: Add lightos support

### PR DESCRIPTION
**Pull request by bennyz**
* Extract the supported drivers tuple to a variable to make it easier to add drivers in the future
* Add lightos to the supported drivers

### CODE REVIEW

**Review by nirs — CHANGES_REQUESTED** [review]
Awsome

File: lib/vdsm/storage/managedvolume.py [hunk 1]
```diff
@@ -108,10 +113,10 @@ def attach_volume(sd_id, vol_id, connection_info):
                     multipath_id=attachment.get("multipath_id"))
                 _invalidate_lvm_devices(attachment)
                 volume_type = connection_info["driver_volume_type"]
-                if volume_type not in ("rbd", "iscsi"):
+                if volume_type not in supported_drivers:
                     raise se.UnsupportedOperation(
-                        "Unsupported volume type, supported types are: "
-                        "rbd, iscsi")
+                        f"Unsupported volume type, supported types are: "
+                        "{supported_drivers}")
```
Comment (aesteve-rh) [inline lib/vdsm/storage/managedvolume.py:123]: Does this work? I'd say the second line of the string needs the `f` to be an f-string. The first line does not need it, but I think looks better if we keep it for consistency.

Comment (bennyz) [reply lib/vdsm/storage/managedvolume.py:123]: yeah, fixed it locally

**Review by aesteve-rh — DISMISSED** [review]

**Review by nirs — DISMISSED** [review]

File: lib/vdsm/storage/managedvolume.py [hunk 2]
```diff
@@ -58,9 +58,14 @@
 DEV_RBD = "/dev/rbd"
 VOLUME_LINK_DIR = "/run/vdsm/managedvolumes/"
 
+# Drivers that have been tested and are known to work.
 SUPPORTED_DRIVERS = (
+    # Drivers tested by the oVirt team
     "rbd",
     "iscsi",
+
+    # Tested by Muli Ben-Yehuda
```
Comment (muliby-lb) [inline lib/vdsm/storage/managedvolume.py:67]: feel free to also add a contact email address, say `<info (at) lightbitslabs.com>`

**Review by muliby-lb — APPROVED** [review]
Awesome!

**Pull request merged**